frontend/pages/hosts/details/cards/Software/DeviceSoftwareTableConfig.tsx

```diff
@@ -55,7 +55,8 @@ export const generateSoftwareTableHeaders = (): ISoftwareTableConfig[] => {
       sortType: "caseInsensitive",
     },
     {
-      Header: "Version",
+      Header: "Installed version",
```

Comment:
yay or nay

Reply:
(note this is for the Fleet desktop > Software > Version header which I don't think shows installers at all so there might not be confusion